R/data-set.R

```diff
@@ -368,12 +364,13 @@ DataSet <- R6::R6Class(
       return(DataRepository$new(dataRepository))
     },
     .initializeCache = function() {
+      dataRepositoryTask <- getNetTask("DataRepositoryTask")
       private$.xColumn <- private$.dataRepository$baseGrid
-      # We assume for now that the first column not base grid in the data column
-      private$.yColumn <- private$.dataRepository$allButBaseGrid[[1]]
 
-      dataRepositoryTask <- getNetTask("DataRepositoryTask")
-      netYErrorColumn <- rClr::clrCall(dataRepositoryTask, "GetErrorColumn", private$.yColumn$ref)
+      netYColumn <- rClr::clrCall(dataRepositoryTask, "GetMeasurementColumn", private$.dataRepository$ref)
+      private$.yColumn <- DataColumn$new(netYColumn)
```

Comment:
The one issue here. The instance of DataColumn won't be in the cache in the DataRepository. I do not think that this is an issue as the ref are the same...

Reply:
No, don't think it will be a problem.